src/sql-parser/src/parser.rs

```diff
@@ -2848,7 +2848,38 @@ impl<'a> Parser<'a> {
             BYTES => Format::Bytes,
             _ => unreachable!(),
         };
-        let include_headers = self.parse_keywords(&[INCLUDE, HEADERS]);
+
+        let mut include_headers = CreateWebhookSourceIncludeHeaders::default();
+        while self.parse_keyword(INCLUDE) {
+            match self.expect_one_of_keywords(&[HEADER, HEADERS])? {
+                HEADER => {
+                    let header_name = self.parse_literal_string()?;
+                    self.expect_keyword(AS)?;
```

Comment:
That's a bit of a recipe for frustration as header names almost always contain `-` characters which are not valid bare identifiers. Maybe we should have a rule that the default name for a header is the header name with all dashes replaced with underscores? `content-encoding` becomes `content_encoding`, for example.

Reply:
Also header names can start with a digit apparently, and valid SQL identifiers cannot!

Reply:
Given the possible complexity because of the constraints on what a valid SQL identifier is, I'd prefer to do this as a follow up? Relaxing the requirement for `AS <column-name>` should be backwards compatible change

Reply:
I think that's a good call.